dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php

```diff
@@ -51,7 +51,7 @@ public function startTestSuite()
     {
         if (null === $this->reader) {
             $this->reader = Bootstrap::getObjectManager()->get(\Magento\Framework\App\DeploymentConfig\Reader::class);
-            $this->config = $this->reader->load();
+            $this->config = $this->filterIgnoredConfigValues($this->reader->load());
```

Comment:
Hello, maybe I'm missing something but where the `filterIgnoredConfigValues()` method comes from?

Reply:
Hi @aleron75 it's on line 90 in that same file. It is only available on the 2.2 releases, and was included in version 2.2.8.